src/views/examples/Profile.js

```diff
@@ -34,20 +28,55 @@ import {
 
 // core components
 import UserHeader from "components/Headers/UserHeader.js";
+import axios from 'axios';
+import Loader from "react-loader-spinner";
 
-const Profile = (props) => {
+const Profile = () => {
 
-  const [resumelink,setResumelink]=useState(props.resumeLink);
-  const [githublink,setGithublink]=useState(props.githubLink);
-  const [linkedinlink,setLinkedinlink]=useState(props.linkedinLink);
+  const [profiledata,setProfileData]=useState([]);
+  const [isLoading,setIsLoading]=useState(true);
 
-  const printdata = (event)=>{
-    console.log({resumelink},{githublink},{linkedinlink});
-    event.preventDefault();
+  const fetchProfile = async() => {
+    const profileDetails= await axios.get('http://127.0.0.1:8000/user/profile/')
+    setProfileData(profileDetails.data)
+    setMasterResumelink(profiledata.mastercv)
+    setResume1link(profiledata.resume1)
+    setResume2link(profiledata.resume1)
+    setGithublink(profiledata.github)
+    setLinkedinlink(profiledata.linkedin)
+    setIsLoading(false)
+
+    
+  }
+  const updateProfile = () => {
+    axios.post('http://127.0.0.1:8000/user/edit/',{
+        mastercv: masterresumelink,
+        resume1: resume1link,
+        resume2: resume2link,
+        github: githublink,
+        linkedin: linkedinlink,
+
+    })
+    
+    
   }
+
+  useEffect(()=>{
+    fetchProfile()
+  },[])
+
+  const [masterresumelink,setMasterResumelink]=useState(profiledata.mastercv);
+  const [resume1link,setResume1link]=useState(profiledata.resume1);
+  const [resume2link,setResume2link]=useState(profiledata.resume2);
+  const [githublink,setGithublink]=useState(profiledata.github);
+  const [linkedinlink,setLinkedinlink]=useState(profiledata.linkedin);
+
+
   return (
+    isLoading?
```

Comment:
use this {isLoading ?    :  }  curly brackets are missing kindly add these.

Reply:
All suggested changes have been made.